tests/dataset_test.py

```diff
@@ -0,0 +1,141 @@
+from qri import dataset
+from qri import loader
+import mock_loader
+import unittest
+
+# TODO - move these to a better place?
+LIST_OBJ = {
+    'username': 'me',
+    'profileID': 'Qm.profileID',
```

Comment:
Please remove the "." from this. ProfileIDs use [base58 encoding](https://en.bitcoin.it/wiki/Base58Check_encoding), for which "." is an illegal character. I know it's just mock data, but I don't want to cause any confusion if someone were to come along in the future, read this, and get a misunderstanding of valid profileIDs.

The same is true for ipfs paths (after the "/" slash) and bodypaths.